buildenv/jenkins/Build-JDK8-linux_390-64_cmprssptrs

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+ * Copyright (c) 2017, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+ node('390') {
+
+    SDK_VERSION = "8"
+    OPENJDK_REPO = "https://github.com/ibmruntimes/openj9-openjdk-jdk${SDK_VERSION}.git"
+    OPENJDK_BRANCH = "openj9"
+    BOOT_JDK = "/usr/lib/jvm/java-7-openjdk-s390x"
+    FREEMARKER = "$HOME/openj9/freemarker.jar"
+    SDK_PREFIX = "OpenJ9-JDK${SDK_VERSION}-linux_390-64_cmprssptrs-"
+    SDK_SUFFIX = ".tar.gz"
+    TEST_PREFIX = "test-"
+    TEST_SUFFIX = ".tar.gz"
+    RELEASE = "linux-s390x-normal-server-release"
+    JDK_FOLDER = "j2sdk-image"
+
+    checkout scm
```

Comment:
This whole section of commands is duplicated in each build file.  Is there a way to remove the duplication?  Or is there some benefit to having it?

On the surface, a single `build_jdk8(${WORKSPACE})` call would be cleaner.